src/test/fs_tests.cpp

```diff
@@ -52,6 +52,18 @@ BOOST_AUTO_TEST_CASE(fsbridge_fstream)
         file >> input_buffer;
         BOOST_CHECK_EQUAL(input_buffer, "bitcoin");
     }
+    {
+        // Join an absolute path and a relative path.
```

Comment:
In commit "Introduce fsbridge::AbsPathJoin(base, p)." (c0da7456833fac6d5613d97e09266cf54f13594f)

It would be good to add another test joining an absolute path with the empty `""` path and ensuring the absolute path is returned unchanged (no trailing slash). This is the difference between boost and std join functions and cause of bugs in #20744